docs/api/README.md

```diff
@@ -6,4 +6,5 @@
 !!!include(docs/api/renderToString.md)!!!
 !!!include(docs/api/selectors.md)!!!
 !!!include(docs/api/createLocalVue.md)!!!
+!!!include(docs/api/createWrapper.md)!!!
```

Comment:
It is necessary to add link to below page.
https://github.com/vuejs/vue-test-utils/blob/8db502dfacf56f5cfc8a914da192f8a43dd5a162/docs/README.md